cmd/testdata/valid_config.yaml

```diff
@@ -1 +1,3 @@
 app-name: hello
+write:
+  create-empty-file: true
```

Comment:
Files should have a blank line at the end. Please add.

Reply:
Will do this in the follow up PR with file cache changes.